plugins/modules/swim_intent.py

```diff
@@ -417,8 +417,9 @@ def validate_input(self):
         """
 
         if not self.config:
-            self.msg = "config not available in playbook for validattion"
             self.status = "success"
+            self.msg = "config not available in playbook for validation"
```

Comment:
self.msg = "Configuration is not available in the playbook for validation"

Reply:
added.